### PR TITLE
Remove Bg Color from block on Merge Paste

### PR DESCRIPTION
**Pull request by BryanValverdeU**
Remove the background color of the source blocks when the mergeType is keepSourceEmphasisFormat.

### CODE REVIEW

File: packages-content-model/roosterjs-content-model-core/lib/publicApi/model/mergeModel.ts [hunk 1]
```diff
@@ -384,6 +386,16 @@ function applyDefaultFormat(
     });
 }
 
+function mergeBlockFormat(applyDefaultFormatOption: string, block: ContentModelBlock) {
+    if (
+        applyDefaultFormatOption == 'keepSourceEmphasisFormat' &&
+        (block.blockType != 'BlockGroup' || block.blockGroupType != 'ListItem') &&
```
Comment (JiuqingSong) [inline packages-content-model/roosterjs-content-model-core/lib/publicApi/model/mergeModel.ts:392]: why keep background color for list item?

Comment (BryanValverdeU) [reply packages-content-model/roosterjs-content-model-core/lib/publicApi/model/mergeModel.ts:392]: For list item we do not have background color in the format interface. 
I think we never parse it, but let be double check.

Comment (BryanValverdeU) [reply packages-content-model/roosterjs-content-model-core/lib/publicApi/model/mergeModel.ts:392]: Yes for ContentModelListItemFormat, we are not extending BackgroundColorFormat.

Comment (JiuqingSong) [reply packages-content-model/roosterjs-content-model-core/lib/publicApi/model/mergeModel.ts:392]: How about we add BackgroundColorFormat into ContentModelListItemFormat?

Comment (BryanValverdeU) [reply packages-content-model/roosterjs-content-model-core/lib/publicApi/model/mergeModel.ts]: Added BackgroundColor Format to ContentModelListItemFormat as suggested.

**Review by JiuqingSong — APPROVED** [review]

**Pull request merged**